content/docs/command-reference/fetch.md

```diff
@@ -17,22 +17,17 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+Downloads tracked data from remote storage into the <abbr>cache</abbr> of the
+project (without placing it in the <abbr>workspace</abbr> like `dvc pull`). This
+makes them available for linking (or copying) into the workspace (refer to
+`dvc config cache.type`).
 
-Without arguments, `dvc fetch` ensures that the files specified in all
-`dvc.lock` and `.dvc` files in the workspace exist in the cache. The
-`--all-branches`, `--all-tags`, and `--all-commits` options enable fetching data
-for multiple Git commits.
+> Note that fetching data does not affect code or
```

Comment:
Is it actually needed? If yes, let's simplify it ... affect is also not the right word here to my mind

Reply:
This is another clarification that "DVC is not Git"... I don't think we should need these but who knows... I guess it's there just in case.

Reply:
Fixed it for now.